src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
+    if (robotValues.indexOf(robotValues[i])
+      !== robotValues.lastIndexOf(robotValues[i])) {
+      return null;
+    }
+  }
+
+  const robotEntries = Object.entries(robot);
+
+  robotEntries.map(el => el.reverse());
```

Comment:
amazing solution! just do it in right way. map is not mutating the array, it returns new one
```suggestion
  const reversedEntries = Object.entries(robot).map(el => el.reverse());
```